cmd/image-loader/main.go

```diff
@@ -131,6 +134,30 @@ func main() {
 		log.Fatalw("Error loading versions", zap.Error(err))
 	}
 
+	// if no local addons path is given, use the configured addons
+	// Docker image and extract the addons from there
+	if o.addonsPath == "" {
+		addonsImage := o.addonsImage
+		if addonsImage == "" {
+			if kubermaticConfig == nil {
```

Comment:
Why you bother checking the kubermatic config? now that we have an explicit argument you could just use the default image if not specified otherwise.

Reply:
Because I want the users in the future to simply give us their KubermaticConfiguration and we figure out everything from there. The addons-path, addons-image etc. flags are only for customers who need to override it.